public/components/SponsorshipEdit/SponsorshipPackageEdit.react.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export default class SponsorshipPackageEdit extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    updateType(e) {
+        this.props.updateSponsorship(Object.assign({}, this.props.sponsorship, {
+            sponsorshipPackage: e.target.value
+        }));
+    }
+
+
+    render () {
+
+        if (!this.props.sponsorship) {
+            return false;
```

Comment:
~if you want it to render nothing, I think returning null is better to avoid `false` being printed to the screen~
```suggestion
            return null;
```

Well, TIL https://medium.com/@davidkelley87/stop-using-return-null-in-react-a2ebf08fc9cd

Reply:
So I should stick with `return false`? Most of this was copy+pasted from `SponsorshipPackageType.react.js`